crates/router/src/core/payments/helpers.rs

```diff
@@ -3494,7 +3507,10 @@ pub fn get_attempt_type(
                 metrics::MANUAL_RETRY_REQUEST_COUNT.add(
                     &metrics::CONTEXT,
                     1,
-                    &add_attributes([("merchant_id", payment_attempt.merchant_id.clone())]),
+                    &add_attributes([(
+                        "merchant_id",
+                        payment_attempt.merchant_id.get_string_repr().to_owned(),
```

Comment:
We can directly pass the `MerchantId` type here (and in other usages in this file), right?